### ISSUE TITLE
[BUG] Novel updates not working

### ISSUE THREAD

**Issue by Kevvvn** [Flameish/Novel-Grabber#196]
**Describe the bug**
When adding a series from novel updates it gets stuck trying to load the chapter list. Also when I try clicking any urls on the site using the browser I get stuck on an endless loop of cloudflare captcha.

**Platform**
Java version, Win10

**Logs**
```[INFO]Starting browser...
577 [pool-2-thread-1] INFO io.github.bonigarcia.wdm.WebDriverManager - Using msedgedriver 90.0.818.66 (resolved driver for Edge 90)
593 [pool-2-thread-1] INFO io.github.bonigarcia.wdm.WebDriverManager - Exporting webdriver.edge.driver as C:\Users\kevv\.cache\selenium\msedgedriver\win64\90.0.818.66\msedgedriver.exe
Starting MSEdgeDriver 90.0.818.66 (5f913625aa6fc53d48c21e7d1fd16b222d2f2aeb) on port 30708
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping MSEdgeDriver safe.
MSEdgeDriver was started successfully.
May 22, 2021 6:41:39 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
[ERROR]Something went wrong, see log for more information.
org.openqa.selenium.TimeoutException: Expected condition failed: waiting for presence of element located by: By.cssSelector: div#my_popupreading ol.sp_chp li a[href] (tried for 30 second(s) with 500 milliseconds interval)
        at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
        at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
        at grabber.sources.novelupdates_com.getChapterList(novelupdates_com.java:62)
        at grabber.Novel.check(Novel.java:81)
        at gui.GUI.lambda$null$0(GUI.java:259)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"div#my_popupreading ol.sp_chp li a[href]"}
  (Session info: MicrosoftEdge=90.0.818.66)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'LASTORDER', ip: '192.168.56.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_281'
Driver info: org.openqa.selenium.edge.EdgeDriver
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 90.0.818.66, javascriptEnabled: true, ms:edgeOptions: {debuggerAddress: localhost:58354}, msedge: {msedgedriverVersion: 90.0.818.66 (5f913625aa6fc5..., userDataDir: C:\Users\kevv\AppData\Local...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: cdfb3d002b14c74341399a9076a6569e
*** Element info: {Using=css selector, value=div#my_popupreading ol.sp_chp li a[href]}
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
        at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
        at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
        at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
        at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
        at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
        at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
        at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:420)
        at org.openqa.selenium.By$ByCssSelector.findElement(By.java:431)
        at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
        at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:182)
        at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:179)
        at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
        ... 6 more```

**Comment by Flameish** [comment]
See #166. Nothing I can really do about it.

**Comment by Flameish** [comment]
Alternatively, you can always go directly to the actual host and use manual grabbing (if the host is not already supported).

**Comment by Flameish** [comment]
Closed in favor of #135
